src/components/LanguageSelector/LanguageSelectorButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import classnames from 'classnames'
+
+type LanguageSelectorButtonProps = {
+  label: string
+  labelAttr?: string
+  isOpen?: boolean
+  onToggle: () => void
+}
+
+export const LanguageSelectorButton = ({
+  label,
+  labelAttr,
+  isOpen,
+  onToggle,
+  className,
+  ...buttonProps
+}: LanguageSelectorButtonProps &
+  JSX.IntrinsicElements['button']): React.ReactElement => {
+  const classes = classnames('usa-button', 'usa-language__link', className)
+  const labelNotEn = labelAttr && <span lang={labelAttr}>{label}</span>
```

Comment:
Since you removed the `curLangNotEn` variable in `LanguageSelector` and are no longer using the `labelAttr` prop from there, do we still need this `labelNotEn` variable? Looking at the test coverage, the `<span>` here is not checked, and it just made me wonder if it is still needed.

Reply:
I'd still keep it on the button so that the button contents get the same lang attr treatment. I just updated it to handle that better if you want to see those changes